packages/app/src/components/inspector/TextStyleInspector.tsx

```diff
@@ -0,0 +1,377 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+
+import {
+  StretchHorizontallyIcon,
+  StretchVerticallyIcon,
+  SquareIcon,
+  TextAlignCenterIcon,
+  TextAlignJustifyIcon,
+  TextAlignLeftIcon,
+  TextAlignRightIcon,
+  PinBottomIcon,
+  PinTopIcon,
+  AlignCenterVerticallyIcon,
+  LetterCaseLowercaseIcon,
+  LetterCaseCapitalizeIcon,
+  LetterCaseUppercaseIcon,
+} from '@radix-ui/react-icons';
+import {
+  Label,
+  Select,
+  RadioGroup,
+  InputField,
+  LabeledElementView,
+  Divider,
+} from 'noya-designsystem';
+import { Spacer } from 'noya-designsystem';
+import { useCallback, memo, useState } from 'react';
+import ColorInputFieldWithPicker from './ColorInputFieldWithPicker';
+
+import * as InspectorPrimitives from './InspectorPrimitives';
+
+interface TextStyleRowProps {
+  fontFamily: string;
+  fontWeight: string;
+  fontColor: Sketch.Color;
+  onChangeFontFamily: (value: string) => void;
+  onChangeFontWeight: (value: string) => void;
+  onChangeFontColor: (color: Sketch.Color) => void;
+}
+
+interface TextAligmentRowProps {
+  fontAlignment: string;
+  fontHorizontalAlignment: string;
+  fontVerticalAlignment: string;
+  onChangeFontAlingment: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onChangeFontHorizontalAlignment: (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => void;
+  onChangeFontVerticalAlignment: (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => void;
+}
+
+interface TextOptionsRowProps {
+  fontCase: string;
+  fontDecorator: string;
+  onChangeFontCase: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onChangeFontDecorator: (value: string) => void;
+}
+
+const TextStyleRow = memo(function TextStyleRow({
+  fontColor,
+  fontFamily,
+  fontWeight,
+  onChangeFontColor,
+  onChangeFontFamily,
+  onChangeFontWeight,
+}: TextStyleRowProps) {
+  const id = '1';
```

Comment:
This number `id` was only need for fills/borders/shadows because they're used in an array. We can delete this